recipe/meta.yaml

```diff
@@ -9,49 +9,47 @@ package:
 source:
   url: https://gitlab.com/stuko/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-  - gcc_7_2.patch
 
 build:
-  number: 5
-  skip: True  # [not linux]
+  number: 0
+  ignore_run_exports:
+    - libxslt
 
 requirements:
   build:
-    - {{ compiler('cxx') }}
     - {{ compiler('c') }}
-    - {{ cdt('mesa-libgl-devel') }}  # [linux]
-    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ compiler('cxx') }}
     - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
     - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxext-devel') }}  # [linux]
     - {{ cdt('libxxf86vm') }}  # [linux]
-    - sphinx
+    - {{ cdt('libXau') }}  # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
     - cmake
-    - make
-    - git
+    - make  # [unix]
   host:
-    - xorg-libxfixes  # [linux]
+    - boost
+    - ffmpeg
+    - libnetcdf
     - qt
-    - python
-    - numpy
-    - pyqt
-    - qscintilla2
-    - boost =1.57.0
+    - libxslt
     - zlib
-    - fftw
   run:
-    - python
-    - matplotlib
-    - ase
+    # nettle from defaults is not compatible
+    - {{ pin_compatible('nettle', exact=True) }}  # [unix]
+
 
 test:
-  imports:
-    - ovito
+  commands:
+    - command -v ovito  # [unix]
+    - where prismatic  # [win]
```

Comment:
I do not understand why you check for prismatic should not it be just a check for Ovito ?

Reply:
The reason is simply: me multitasking in the evening and doing this mistake! ;) As silver lining, it confirms that this test fails when the executable is not installed! :D